packages/@glimmer/interfaces/lib/managers/modifier.d.ts

```diff
@@ -19,4 +19,6 @@ export interface ModifierManager<ModifierStateBucket> {
   installModifier(instance: ModifierStateBucket, element: Element, args: Arguments): void;
   updateModifier(instance: ModifierStateBucket, args: Arguments): void;
   destroyModifier(instance: ModifierStateBucket, args: Arguments): void;
+  getDebugName?(instance: object): string;
```

Comment:
Not sure if instance is just misnamed here, but if that is named correctly then it doesn't seem good, we shouldn't need to have created the instance just to get the name, we want to be able to use this statically (on the class, for error messages, etc)

Reply:
now its ModifierStateBucket